media/Circletracer/loadfile.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+let jsonLoadCheck = 0;
```

Comment:
It would be better to change this as function.
For example,

```javscript
function loadedJsonCnt() {
  var cnt = 0;
  if (document.querySelector('#first-json-btn').disabled === true)
    ++cnt;
  if (document.querySelector('#second-json-btn').disabled === true)
    ++cnt;
  return cnt;
}
```

Reply:
Looking at it now, there's no need to declare that variable in static area.
Thank u :)

Reply:
The reason why the variable is located static is that it is also used in other functions.

ex) `reset, processData`

Reply:
Also, it is recommended to use the`let` rather than `var` because of hosting.

Reply:
Usually, using global/static variable is not that recommended.
So I recommended to change the usage of `jsonLoadCheck` to `loadedJsonCnt` in other functions :)

Reply:
Thank u :)

Reply:
I reflected everything you said. Can you check it again? :)